lte/gateway/deploy/roles/dev_common/tasks/main.yml

```diff
@@ -272,7 +272,7 @@
 
 - name: Install Sentry Native SDK
   vars:
-    sentry_native_version: "0.4.8"
+    sentry_native_version: "0.4.9"
```

Comment:
currently we have separate ways to install sentry in production and dev environment. It would be nice to use same mechanism for both. Can you add it ad TODO.

Reply:
how do we install sentry for prod? @pshelar 
Yeah will add a todo

Reply:
package is built using: https://github.com/magma/magma/blob/master/third_party/build/bin/sentry-native-build.sh
This is deployed in production. we could deploy same in vagrant VM.